web/styles/typing_notifications.css

```diff
@@ -9,3 +9,7 @@
     list-style: none;
     margin: 0;
 }
+
+.align_below_mark_as_read_banner {
+    margin-top: -18px;
+}
```

Comment:
Not sure if this is the best place to have this rule here

Reply:
Using a negative margin for this feels a bit off for me. We could use that if we don't have any other solutions, sure. But could you investigate why we have that margin below the "Mark as read" banner?

Reply:
Okay, while this margin is important for the compose box, I don't see its purpose in the main message area, we should override it there. That should remove the need for negative margins.

Reply:
updated!